docker-hub/api/latest.yaml

```diff
@@ -1,13 +1,7 @@
-swagger: '2.0'
```

Comment:
Doing a major upgrade here breaks our current generation of some APIs and what is copied into here. So each time a change occurs in an API we then have to translate from swagger 2 to 3.